src/style.css

```diff
@@ -0,0 +1,452 @@
+/* stylelint-disable csstree/validator */
```

Comment:
- [ ] **[OPTIONAL]** For this one I see that the rule is complaining about the usage of `clamp()` as a value for the `font-size` property. I did some research, and it looks like this function is not supported by the version of the linter. 📜 My suggestion when cases like this happen (e.g. a matter of versioning) is to add a short comment next to the disabled rule explaining what are the reasons to disable it, so people reviewing your code can understand this decision! 🤓 🔧